apps/studio/src/store/index.ts

```diff
@@ -435,6 +448,9 @@ const store = new Vuex.Store<State>({
           onlyTables.forEach((t) => {
             t.entityType = 'table'
             t.columns = []
+            t.partitions = []
+            if (context.state.connection.connectionType != 'postgresql') 
```

Comment:
I don’t like this connection type awareness living here. Can we check supportedFeatures or have listTables set this to mull?